ui/src/components/SampleView/ContextMenu.jsx

```diff
@@ -430,18 +402,14 @@ export default class ContextMenu extends React.Component {
         }
       });
     }
-
-    this.props.sampleViewActions.showContextMenu(false);
```

Comment:
All of these are replaced with Bootstrap's Dropdown's `autoClose` feature.

Reply:
nice !